src/modelbench/run.py

```diff
@@ -76,26 +76,33 @@ def cli() -> None:
     multiple=True,
 )
 @click.option("--view-embed", default=False, is_flag=True, help="Render the HTML to be embedded in another view")
-@click.option("--anonymize", type=int, help="Random number seed for consistent anonymization of SUTs")
-@click.option("--parallel", default=False, help="experimentally run SUTs in parallel")
+@click.option(
+    "--custom-branding",
+    type=click.Path(file_okay=False, dir_okay=True, path_type=pathlib.Path),
+    help="Path to custom branding. Implicitly sets --generic-report to remove MLCommons copy.",
+)
 @click.option(
```

Comment:
Would it make more sense here to just have one CLI option that is either 1) `--custom-branding` which includes MLC branded stuff or 2) no `--custom-branding` which is always generic?

Reply:
Wouldn't users need to know the path of MLC branding in that case?

Reply:
I think for now the only users running reports with MLC branding will be the three of us. And we'll eventually put that branding directory in `modellab`, where we'll be running the official reports from.